examples/quickstart/jupyter-notebooks/api-tutorial.ipynb

```diff
@@ -0,0 +1,442 @@
+    "  \"context\": {\n",
+    "    \"maxNumTasks\": 3\n",
+    "  }\n",
+    "})\n",
+    "\n",
+    "headers = {'Content-Type': 'application/json'}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "ingestiion_taskId_response = response\n",
+    "print(response.text + f\"\\nInserting data into the table named {dataSourceName}.\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1235e99-be72-40b0-b7f9-9e860e4932d7",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Extract the `taskId` value from the `taskId_response` variable so that you can reference it later:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f578b9b2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ingestion_taskId = json.loads(ingestiion_taskId_response.text)['taskId']\n",
+    "print(ingestion_taskId)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f17892d9-a8c1-43d6-890c-7d68cd792c72",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Get the status of your task\n",
+    "\n",
+    "The following cell shows you how to get the status of your ingestion.  You can see basic information about your query, such as when it started and whether or not it's finished.\n",
+    "\n",
+    "In addition to the status, you can retrieve a full report about it if you want using `GET /druid/indexer/v1/task/TASK_ID/reports`. But you won't need that information for this tutorial."
```

Comment:
Do we need to indicate that you need status of `SUCCESS`  to move forward in the tutorial?

How hard would it be to write a loop that checks every 30 seconds for `SUCCESS` that way this block will keep you from moving forward if the datasource isn't ready to accept queries

Reply:
We could. It could be something as simple as 
while the response is not success:
* sleep for 10 seconds
* run the query again

But I think that's a bit of an overkill though since the dataset is so small.